apps/docs/content/guides/cli/seeding-your-database.mdx

```diff
@@ -50,12 +50,42 @@ COPY countries (name, code) FROM 'seed/countries.csv' WITH (FORMAT csv);
 
 ## Generating seed data
 
-You can generate seed data for local development using [Snaplet](https://www.snaplet.dev/).
+You can generate seed data for local development using [Snaplet](https://github.com/snaplet/seed).
 
-If this is your first time using Snaplet to seed your project, you'll need to setup Snaplet
-for your project with `npx @snaplet/seed init`, and follow the interactive prompts. The command
-will look at your database and its structure, and use it to generate the configuration files
-and assets needed by `@snaplet/seed`.
+<Admonition type="tip">
+  To use Snaplet, you need to have Node.js and npm installed. You can add Node.js to your project by
+  running `npm init -y` in your project directory.
+</Admonition>
```

Comment:
```suggestion
<Admonition type="tip">

To use Snaplet, you need to have Node.js and npm installed. You can add Node.js to your project by running `npm init -y` in your project directory.

</Admonition>
```

Adding the blank lines will prevent Prettier from doing the odd line-break thing (which shows up in the final output).